src/backend/main/go.main/facilities/Notifications/irc.go

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
I'm not sure we will use irc as notifier channel.

Then if not implemented then not put it in code, for all protocols

Reply:
ok commit #ad6494e